playbooks/roles/common/tasks/main.yml

```diff
@@ -0,0 +1,50 @@
+- name: Copy hosts file
```

Comment:
@Agrendalath, it looks like most, if not all of this is already covered in the `common-server-init` role. Can we just reuse that and only add whatever is missing in that?